PVSSpringBoot/src/test/java/com/example/PVSSpringBoot/RepoTest/OAuthTest.java

```diff
@@ -0,0 +1,67 @@
+package com.example.PVSSpringBoot.RepoTest;
+import com.example.PVSSpringBoot.OAuth2.AOuth2Service;
+import com.example.PVSSpringBoot.repositories.UsersRepo;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
+@ExtendWith(SpringExtension.class)
+public class OAuthTest {
+
+    @Autowired
+    private UsersRepo usersRepo;
+
+    @Autowired
+    public void UserRepoTest(UsersRepo usersRepo) {
+        this.usersRepo = usersRepo;
+    }
+    @AfterEach
+    public void wipeDB(){
+        usersRepo.deleteAll();
+    }
+
+// -------Test for Login for User already in database
+@Test
+public void Test_for_Login_for_User_already_in_data_base() {
+    AOuth2Service aouth2Service = new AOuth2Service(this.usersRepo);
+    String res = aouth2Service.LogInOAuthUser("abdelrahmanelsayd223@gmail.com","abdelrahman","elsayd");
```

Comment:
Be careful to remove the user name from the test after removing them from AOuth2Service.java file